pkg/controller/operators/olm/apiservices.go

```diff
@@ -830,19 +888,25 @@ func (a *Operator) installAPIServiceRequirements(desc v1alpha1.APIServiceDescrip
 		}
 
 		if !adoptable {
-			return nil, fmt.Errorf("pre-existing APIService %s is not adoptable", apiServiceName)
+			return fmt.Errorf("pre-existing APIService %s is not adoptable", fmt.Sprintf("%s.%s", desc.Version, desc.Group))
```

Comment:
```suggestion
			return fmt.Errorf("pre-existing APIService %s.%s is not adoptable", desc.Version, desc.Group)
```

Reply:
Accepted in the latest version of the PR.